src/apt_package_function/func_app.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Management of function applications"""
```

Comment:
did you consider using azure sdk and /or azure-functions library?

Reply:
I did but pulling in the Azure SDK in pure Python is several hundreds of megabytes and dependencies. I might be able to use the management libraries more directly but this is a good stopgap.